### ISSUE TITLE
Добавить вывод графиков к скрипту обучения

### ISSUE THREAD

**Issue by serykhelena** [lsd-maddrive/mnist_recognition#41]
Чтобы более наглядно следить за тем, что происходит с моделью в процессе обучения - удобно использовать графики. 

`from torch.utils.tensorboard import SummaryWriter` - https://pytorch.org/docs/stable/tensorboard.html
Нужно в твой скрипт обучения встроить эту штуку, в результате ты должна получить следующие графики: 
- функции потерь при train 
- функции потерь при valid 
- Изменение learning rate (здесь есть маленькая хитрость, нужно погуглить :smiley_cat: ) 

> Модуль tensorboard будет рисовать графики тебе в браузер в реальном времени, при желании их можно будет сохранить себе локально. Но главное, что это наглядно, а трудозатрат минимально (=
